content/en/_index.html

```diff
@@ -19,7 +19,7 @@
                 <div class="col-sm-10 col-sm-offset-1 col-xs-12">
                     <h2 class="section--title">What we do</h2>
                     <p class="text-left">We are focused on delivery: helping government design and build better services.</p>
-                    <h3>We are partnering to...</h3>
+                    <h3>We are partnering to hello...</h3>
```

Comment:
Need to remove hello :)

Reply:
oh goodness, sorry about that! fixed it !